web_tree_many2one_clickable/__manifest__.py

```diff
@@ -2,18 +2,20 @@
 # Copyright 2015 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
 # Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
 # Copyright 2017 Sodexis <dev@sodexis.com>
+# Copyright 2024 Versada (https://versada.eu)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Clickable many2one fields for tree views",
     "summary": "Open the linked resource when clicking on their name",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
     "category": "Hidden",
     "website": "https://github.com/OCA/web",
     "author": "Therp BV, "
     "Tecnativa, "
     "Camptocamp, "
     "Onestein, "
+    "Versada, "
```

Comment:
This migration doesn't qualify for being co-author. Just add you as contributor.